src/com/galactanet/gametable/GametableFrame.java

```diff
@@ -2926,12 +2922,14 @@ public void propertyChange(final PropertyChangeEvent e)
 
         // Load frame preferences
         loadPrefs();
- 
+
         setContentPane(new JPanel(new BorderLayout())); // Set the main UI object with a Border Layout
         setDefaultCloseOperation(EXIT_ON_CLOSE);        // Ensure app ends with this frame is closed
         setTitle(GametableApp.VERSION);                 // Set frame title to the current version
         setJMenuBar(getMainMenuBar());                  // Set the main MenuBar
-        
+
+        initialisePlugins();
```

Comment:
We might need to look again at where this is called depending on other plugins needs in the future, but this works for now